src/doc/src/reference/build-scripts.md

```diff
@@ -403,6 +403,12 @@ variables like `TARGET` that [Cargo sets for build scripts][build-env]. The
 environment variables in use are those received by `cargo` invocations, not
 those received by the executable of the build script.
 
+As of 1.46, variables used by [`env!`][env-macro] and 
```

Comment:
Thanks!

Unsure if we want to put these under `> **MSRV:**` quote like other parts of the book. Maybe unnecessary?
WDYT?

Reply:
I am fine with this as-is to be honest. No need to put everything in.